packages/core/src/actors/observable.ts

```diff
@@ -64,14 +67,15 @@ export function fromObservable<T, TInput>(
           });
           return {
             ...state,
-            data: (event as any).data
+            output: (event as any).data
```

Comment:
note: this is a mistake - `nextEventType` shouldn't assign to `output`. I'm working on this branch locally so I'll fix this.